index.js

```diff
@@ -25,13 +27,57 @@ elixir.extend('wiredep', function(config, opts) {
             js: '<script src="{{filePath}}"></script>',
             css: '<link rel="stylesheet" href="{{filePath}}" />'
           }
-        }
+        },
+        less: {
+          block: /(([ \t]*)\/\/\s*bower:*(\S*))(\n|\r|.)*?(\/\/\s*endbower)/gi,
+          detect: {
+              css: /@import\s['"](.+css)['"]/gi,
+              less: /@import\s['"](.+less)['"]/gi
+          },
+          replace: {
+              css: '@import "{{filePath}}";',
+              less: '@import "{{filePath}}";'
+          }
+        },
+        sass: {
+          block: /(([ \t]*)\/\/\s*bower:*(\S*))(\n|\r|.)*?(\/\/\s*endbower)/gi,
+          detect: {
+              css: /@import\s(.+css)/gi,
+              sass: /@import\s(.+sass)/gi,
+              scss: /@import\s(.+scss)/gi
+          },
+          replace: {
+              css: '@import {{filePath}}',
+              sass: '@import {{filePath}}',
+              scss: '@import {{filePath}}'
+          }
+        },
+        scss: {
+          block: /(([ \t]*)\/\/\s*bower:*(\S*))(\n|\r|.)*?(\/\/\s*endbower)/gi,
+          detect: {
+              css: /@import\s['"](.+css)['"]/gi,
+              sass: /@import\s['"](.+sass)['"]/gi,
+              scss: /@import\s['"](.+scss)['"]/gi
+          },
+          replace: {
+              css: '@import "{{filePath}}";',
+              sass: '@import "{{filePath}}";',
+              scss: '@import "{{filePath}}";'
+          }
+        },
```

Comment:
These are the default options for the Wiredep package, there's no need to duplicate them. We just need to keep the PHP option since that one doesn't exists by default